src/Template/AngularTemplates/interface_language.ctp

```diff
@@ -24,7 +24,7 @@ usort(
     }
 );
 
-$languagesJSON = htmlspecialchars(json_encode($languages), ENT_QUOTES, 'UTF-8');
+$languagesJSON = h(json_encode($languages), ENT_QUOTES, 'UTF-8');
```

Comment:
You don’t need to provide the extra parameters `ENT_QUOTES, 'UTF-8'`. That’s the neat thing about `h()` :blush: